nestkernel/sparse_node_array.h

```diff
@@ -60,7 +60,11 @@ class SparseNodeArray
 public:
   struct NodeEntry
   {
-    NodeEntry() = default;
+    NodeEntry()
+      : node_( nullptr )
+      , node_id_( 0 )
+    {
+    }
```

Comment:
Same here, the defaulted constructor will zero-initialize the variables.

Reply:
See above.